documentation/api/query/v4/upgrading-from-v3.markdown

```diff
@@ -24,6 +24,9 @@ Each change below is marked with the corresponding release version.
 
 #### /v4/catalogs
 
+* (3.0) The v4 catalogs endpoint has changed the response of the `edges` and `resources` fields
+  to be expanded. For more information see [/v4/catalogs documentation](./catalogs.html).
```

Comment:
we should also note that the catalogs endpoint will no longer return data for deactivated nodes